### PR TITLE
Remove wrong css class

### PR DESCRIPTION
**Pull request by MikeAlhayek**

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/SelectPart.cshtml [hunk 1]
```diff
@@ -14,14 +14,14 @@
         fieldValue = fieldEntry.AttemptedValue;
         if (fieldEntry.Errors.Count > 0)
         {
-            fieldClass = "form-control input-validation-error";
+            fieldClass += " input-validation-error";
```
Comment (Skrypt) [inline src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/SelectPart.cshtml:17]: if it is a `<select>` form element then it should be "form-select".

**Review by Skrypt — APPROVED** [review]

File: src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/SelectPart.cshtml (hunk 1 above)
Comment (ns8482e) [inline src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/SelectPart.cshtml:17]: string.Format or $

Comment (Piedone) [reply src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/SelectPart.cshtml:17]: Why would that be better? This concatenation is simple, the intent is clear, and it's also shorter than writing the following (and especially using `string.Format()`:

```c#
fieldClass = $"{fieldClass} input-validation-error";
```

Comment (MikeAlhayek) [reply src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/SelectPart.cshtml:17]: Yeah. Not sure either. Unless there is a convincing reason I would rather leave it as is

Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/SelectPart.cshtml:17]: I think the comment is about injecting the extra content when the initial value is created, in a single line. Either way.

**Pull request merged**